kmod/src/ioctl.c

```diff
@@ -516,7 +519,11 @@ static long scoutfs_ioc_stage(struct file *file, unsigned long arg)
 	}
 
 	si->staging = true;
+#ifdef KC_LINUX_BACKING_DEV_INFO
```

Comment:
Can't we always use inode_to_bdi() and drop all the compat and ifdefs?

Reply:
Yes and I had missed that RHEL7 even exports it. so, much simpler now.